### PR TITLE
Fix Gov V2 Benchmarks

### PR DESCRIPTION
**Pull request by shawntabrizi**
Benchmarks for Governance V2 were failing: https://github.com/paritytech/polkadot/pull/5205

This should fix them.

Fixes: https://github.com/paritytech/substrate/issues/11906

### CODE REVIEW

**Comment by ggwpez** [comment]
Does this fix all of them since they are related?  
I recorded three pallets as failing: https://github.com/paritytech/substrate/issues/11906

**Comment by shawntabrizi** [comment]
@ggwpez okay might need to check the other pallets

**Comment by shawntabrizi** [comment]
@ggwpez the other failing pallets are fixed by updating the Polkadot configuration.

File: frame/referenda/src/lib.rs [hunk 1]
```diff
@@ -699,7 +699,8 @@ impl<T: Config<I>, I: 'static> Pallet<T, I> {
 		when: T::BlockNumber,
 	) -> Option<(T::BlockNumber, ScheduleAddressOf<T, I>)> {
 		let alarm_interval = T::AlarmInterval::get().max(One::one());
-		let when = (when + alarm_interval - One::one()) / alarm_interval * alarm_interval;
+		let when = when.saturating_add(alarm_interval).saturating_sub(One::one()) /
+			(alarm_interval.saturating_mul(alarm_interval)).max(One::one());
```
Comment (shawntabrizi) [inline frame/referenda/src/lib.rs:703]: unsafe math here caused a panic in benchmarks

**Review by gavofyork — APPROVED** [review]

**Review by ggwpez — APPROVED** [review]
Should I diener and test again or are you confident that it works?

**Review by KiChjang — APPROVED** [review]

**Comment by shawntabrizi** [comment]
bot merge

**Pull request merged**